dark/dna.py

```diff
@@ -215,3 +216,40 @@ def _ambiguousMatch(a, b, matchAmbiguous):
             'gapOffsets': read2GapOffsets,
         },
     }
+
+
+def findKozakConsensus(read):
+        """
+        In a given DNA sequence, search for a Kozak consensus: (gcc)gccRccATGG.
+        Upper case bases are required, lower case bases are the most frequent
+        bases at the given position. The sequence in brackets is of uncertain
```

Comment:
s/brackets/parentheses/

Reply:
I can't find the mistake - would you use parentheses instead of brackets?